imagetest/fixtures.go

```diff
@@ -58,6 +58,19 @@ type TestVM struct {
 	instance     *daisy.Instance
 }
 
+// TestVMParams stores config params for a test vm
+type TestVMParams struct {
```

Comment:
This has one of the same limitations that that param map did: every option needs to be explicitly added and copied over to the daisy.Instance in appendCreateVMStep. Why not take a daisy.Instance directly and allow the caller to specify any options daisy is capable of understanding?

Reply:
I see, working on a new PR which uses *daisy.Instance directly